Source/WebCore/css/parser/CSSSelectorParser.h

```diff
@@ -34,9 +34,8 @@
 #include "CSSParserSelector.h"
 #include "CSSParserTokenRange.h"
 #include "CSSSelectorList.h"
+#include "CSSSelectorParserContext.h"
```

Comment:
You are only using a reference to it ; so you can just forward declare it, no need to include the header.

I think header should be only used in the code if it can be, lower the risk of compilation time increasing over time like it so often does.